TipCatDotNet.Api/Services/HospitalityFacilities/FacilityService.cs

```diff
@@ -182,7 +182,11 @@ private async Task<Result<SlimFacilityResponse>> GetSlimFacility(int facilityId,
         {
             var facility = await _context.Facilities
                 .Where(f => f.Id == facilityId)
-                .Select(SlimFacilityProjection())
+                .GroupJoin(
```

Comment:
The body of tis method may be replaced with
```csharp
GetSlimFacilities(accountId, cancellationToken)
    .SingleOrDefaultAsync(f => f.Id == facilityId, cancellationToken)
```
to avoid duplication